runtime/near-vm-errors/src/lib.rs

```diff
@@ -232,61 +267,110 @@ pub enum HostError {
     /// Tried to call an empty method name
     EmptyMethodName,
     /// Smart contract panicked
-    GuestPanic { panic_msg: String },
+    GuestPanic {
+        panic_msg: String,
```

Comment:
I think it is better to revert the formatting in this file to make the review less complicated